taskfile/task.go

```diff
@@ -17,6 +17,7 @@ type Task struct {
 	Desc                 string
 	Prompt               string
 	Summary              string
+	Requires             []string
```

Comment:
We'll need to update `docs/static/schema.json` as well to include this new attribute:

```json
"requires": {
  "description": "A list of variable which should be set if this task is to run.",
  "type": "array",
  "items": {
    "type": "string"
  }
}
```

Reply:
I missed this, is this generated in anyway or manually updated?

Reply:
Unfortunately, this is currently written manually. Happy to help out if you're not sure on anything.

Reply:
I've updated the schema. Sheesh! that's hard to navigate/edit and it's clear a range of styles have crept in over time, I'd suggest looking at some generation approach as this project grows, but you probably know that ;)

I've sanity checked the JSON schema using VS Code against a Taskfile using the new keyword/field and it all looked good

Reply:
I definitely agree that it's hard to navigate/edit. You get to know the JSON file once you've done it a few times, so I forget how painful it can be the first time you make changes to it. Schema generation is definitely something I'd like to look at in the future, but depending on the implementation, its probably quite a bit of work.

> a range of styles have crept in over time

Ha yeah, the casing/naming is all over the place.